src/components/Common/ScrollToTop.tsx

```diff
@@ -0,0 +1,41 @@
+import { ArrowUpFromDot } from "lucide-react";
+import { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import { cn } from "@/lib/utils";
+
+import { Button } from "@/components/ui/button";
+
+export function ScrollToTopButton({ className }: { className?: string }) {
+  const { t } = useTranslation();
+  const [isVisible, setIsVisible] = useState(false);
+  useEffect(() => {
+    const onScroll = () => {
+      setIsVisible(window.scrollY > 100);
+    };
+    window.addEventListener("scroll", onScroll);
+    return () => window.removeEventListener("scroll", onScroll);
+  }, []);
+
+  const scrollToTop = () => {
+    window.scrollTo({ top: 0, behavior: "smooth" });
```

Comment:
let css styling take care of it (which already is)

```suggestion
    window.scrollTo({ top: 0 });
```